moabb/evaluations/metasplitters.py

```diff
@@ -0,0 +1,179 @@
+import numpy as np
+from sklearn.model_selection import (
+    BaseCrossValidator,
+    StratifiedShuffleSplit,
+)
+
+
+class OfflineSplit(BaseCrossValidator):
+
+    def __init__(self, n_folds):
+        self.n_folds = n_folds
+
+    def get_n_splits(self, metadata):
+        subjects = len(metadata.subject.unique())
+        sessions = len(metadata.session.unique())
+        return subjects * sessions
+
+    def split(self, X, y, metadata, **kwargs):
+
+        subjects = metadata.subject
+
+        for subject in subjects.unique():
+            X_, y_, meta_ = (
+                X[subjects == subject],
+                y[subjects == subject],
+                metadata[subjects == subject],
+            )
+            sessions = meta_.session.values
+
+            for session in sessions:
+                ix_test = np.nonzero(sessions == session)[0]
+
+                yield ix_test
+
+
+class TimeSeriesSplit(BaseCrossValidator):
+
+    def __init__(self, calib_size):
+        self.calib_size = calib_size
+
+    def get_n_splits(self, metadata):
+        sessions = metadata.session.unique()
+        subjects = metadata.subject.unique()
+
+        splits = len(sessions) * len(subjects)
+        return splits
+
+    def split(self, X, y, metadata, **kwargs):
+
+        runs = metadata.run.unique()
+        sessions = metadata.session.unique()
+        subjects = metadata.subject.unique()
+
+        if len(runs) > 1:
```

Comment:
So if `len(runs)>1` then `calib_size` is ignored. 

Is this a desired behaviour @brunaafl  @bruAristimunha ?

If yes, this must be very clear in the doc.

Reply:
It was what I intended, but I don't know if it is the best solution, thought.